src/.env

```diff
@@ -0,0 +1 @@
+MAPBOX_ACCESS_TOKEN=sk.eyJ1IjoiYXNhbnRvczMwMjYiLCJhIjoiY2oxY293bGk4MDA2NDMzcGRxNGdyc3lpbSJ9.uxcZ_o19abopWFm2vwstmg
```

Comment:
We'd like to avoid committing personal access tokens to the GitHub repo. Developers will need to use their own token with the library.